### PR TITLE
Project Plan Update: Tan Heng Yeow

### PR DESCRIPTION
**Pull request by tanhengyeow**
Hi @wkurniawan07, @tran-tien-dat 

I have updated my project plan. Please review! 😄 

I have 2 questions to clarify:
1. If `instructorCourseDetailsPage` and `instructorStudentListPage` were to adopt the new spreadsheet interface, should we still keep the current `edit` button? Also, I believe we are keeping the `view` button for the student for instructors to view students' profile page, which leads to my next question.
2. I noticed that there is a current project `Front-end migration` ongoing. I observed there won't be potential clashes in the work done for this project and the migration but is there anything I should take note of that will have potential overlaps? I have not explored integrating the supported frameworks (Angular, React, Vue) for Handsontable yet but that might be something I should take note of.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Your plan looks good to me. To answer your questions:

> If `instructorCourseDetailsPage` and `instructorStudentListPage were` to adopt the new spreadsheet interface, should we still keep the current edit button?

I may believe that it is not, but you can try it first and see what works.

> I noticed that there is a current project Front-end migration ongoing. I observed there won't be potential clashes in the work done for this project and the migration but is there anything I should take note of that will have potential overlaps?

Good if you want to do, but don't let it be a roadblock for you. If you're using handsontable, wrappers exist for all of React, Angular, and Vue so you need not worry too much.

File: Tan_Heng_Yeow/progress.md [hunk 1]
```diff
@@ -3,78 +3,140 @@
...
+* Fix issues after integration tests, system tests and acceptance tests.
 
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Finish up the remaining work for this [PR](https://github.com/TEAMMATES/teammates/pull/7568)
+- [ ] Write tests for features in the spreadsheet interface
```
Comment (tran-tien-dat) [inline Tan_Heng_Yeow/progress.md:26]: What's the difference between this point and the feature-specific tests in the following weeks?

Comment (tanhengyeow) [reply Tan_Heng_Yeow/progress.md:26]: There were some functionalities that were implemented for the spreadsheet interface in the PR but no tests were written. This point covers the tests for those functionalities.

**Comment by wkurniawan07** [comment]
@tran-tien-dat any other blocking request?

**Comment by tran-tien-dat** [comment]
Nope. I think the plan is good.

**Review by tran-tien-dat — APPROVED** [review]

**Review by wkurniawan07 — APPROVED** [review]

**Pull request merged**